web/styles/recent_topics.css

```diff
@@ -25,6 +25,14 @@
          */
         min-height: 100vw;
 
+        & table {
+            table-layout: fixed;
+
+            & .unread_sort {
+                width: 74px;
```

Comment:
I'm not really feeling good about setting the width as a constant value in this case. Do you think there's a way we can utilize `flexbox` or let the browser handle it for us? It might make things easier and more flexible.

Reply:
Yeah, I do think we need a better way to handle this. I'll need to experiment more with what works for that. As far as I can remember, this is what worked best over others.

Reply:
Removed.